.env.example

```
@@ -73,12 +73,13 @@ SF_TOKEN_ENCRYPTION_IV=
 SF_TOKEN_ENCRYPTION_SALT=
 
 # leave commented for production
-# you probably need dev+test for dev
-#COMPOSE_FILE=docker-compose.yml:compose.dev.yml:compose.test.yml
+#COMPOSE_FILE=docker-compose.yml:compose.dev.yml
 
 # used in compose.dev.yml
 #DB_EXPOSED_PORT=3306
-# (dev) fill with result of `id -u`
+# fill with result of `id -u`
 #UID=
-# (dev) fill with result of `id -g`
+# fill with result of `id -g`
 #GID=
+# to access Mailpit web interface
+#MAILPIT_WEB=8025
```

Pour une histoire de cohérence avec le reste, je propose de renommer en

```suggestion
#MAILPIT_WEB_PORT=8025
```